src/components/editor/EditAuthorsForm.jsx

```diff
@@ -26,59 +29,74 @@ export default class EditAuthorsForm extends BaseComponent {
   }
 
   render() {
+    const styles = {
+      wrapper: {
+        display: 'flex',
+        flexWrap: 'wrap',
+      },
+    }
+
+    const renderChips = (authors) =>
+      _.map(authors, (author) => {
+        return(
+          <Chip
+            key={author.id}
+            onRequestDelete={this.props.handleDeleteAuthor.bind(null, author.id, true)}
+            style={{margin: 4}}
+          >
+            {author.name}
+          </Chip>
+        )
+      });
+
     return (
-      <div style={this.props.style}>
+      <div>
         {
-          _.map(this.props.authors, (author) => {
-            // Striked will correctly evaluate to false if there is no key
-            // with that slug yet, which means it has neither been striked nor unstriked yet
-            // It will return undefined which is a falsy value
-            let striked = this.props.authorsDeleted[author.id];
-            let authorNameStyle = {marginLeft: "1em"};
-            if (striked) {
-              authorNameStyle.textDecoration = "line-through";
-              authorNameStyle.opacity = 0.5;
-            }
+          // TODO: EMIL: not exactly sure what this code is doing so I didnt want to delete
```

Comment:
I of course can't see how it looks now, but the code below is what allows the deleting of added authors and toggling of deletion of current authors:
![delete-authors](https://cloud.githubusercontent.com/assets/18149242/25062679/5143fff2-2196-11e7-9629-500d73981c2f.gif)